wfl/calculators/utils.py

```diff
@@ -22,31 +22,5 @@ def clean_rundir(rundir, keep_files, default_keep_files, calculation_succeeded):
     else:
         clean_dir(rundir, keep_files, force=False)
 
-def parse_genericfileio_profile_argv(argv):
```

Comment:
What's the deal with this function?  Is it just not used?

Reply:
Yes, they simplified a lot what the "Profile" does. It used to parse for the mpi executable, num tasks, etc, now it's all in one string provided by the user.

Reply:
On a quick look, the parsing is now done in the BaseProfile